TAs/optee_ta/Makefile

```
@@ -1,9 +1,11 @@
 
 export V?=0
 
-.PHONY: all
-all:
+.PHONY: all fTPM authvars
```

Should use consistent casing
```suggestion
.PHONY: all fTPM AuthVars
```

---

Makefile convention for targets is all lowercase, with '_' for seperating words if necessary. I'll make fTPM consistent since it is the odd one out and an artifact of a very old set of Makefiles.